docs/quickstart.md

```diff
@@ -0,0 +1,13 @@
+---
+id: quickstart 
```

Comment:
```suggestion
id: quick-start
```
This would match the style of the others

Reply:
Also changing the folder to `quick-start`, to make it https://manual.avolites.com/docs/quick-start/page

Reply:
I've changed both the folder and the id